lib/rubocop/cop/rspec/factory_bot/static_attribute_defined_dynamically.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Prefer declaring static attribute values without a block.
+        #
+        # @example
+        #   # bad
+        #   kind { :static }
+        #
+        #   # good
+        #   kind :static
+        #
+        #   # bad
+        #   comments_count { 0 }
+        #
+        #   # good
+        #   comments_count 0
+        #
+        #   # bad
+        #   type { User::MAGIC }
+        #
+        #   # good
+        #   type User::MAGIC
+        class StaticAttributeDefinedDynamically < Cop
+          MSG = 'Dont use a block to set a static value to an attribute.'.freeze
+
+          def_node_matcher :block_ast_method, <<-PATTERN
+          (block (send nil? _) _ ($_ ...))
+          PATTERN
+
+          def_node_search :factory_attributes, <<-PATTERN
+          (block (send nil? { :factory :trait } ...) _ { (begin $...) $(send ...) $(block ...) } )
```

Comment:
How this `:trait` is being used? Because it's being skipped in line 39, right?

Reply:
Good question ;)

I'm new to working with the source parsing. This bulk of this was copied verbatim from the `FactoryBot/DynamicAttributeDefinedStatically` cop

Having said that, it would appear that L39 is the one that should be removed (similarly for the same line in the dynamic attribute cop!)